src/modules/i18n/locales/en.json

```diff
@@ -172,6 +172,7 @@
     "draw_type_marker": "Line / surface",
     "drawing_attached": "Drawing added as attachment",
     "drawing_empty_cannot_edit_name": "Please add something to the drawing before editing its name.",
+    "drawing_not_shared_admin_warning": "Attention, you have not generated a link to continue to edit your drawing. You won't be able to edit your drawing after closing this window.",
```

Comment:
@Luke252 @hansmannj For all translation I would say `after reloading the website` instead of `closing this windows` (or something like this), because it is more correct, you can close the drawing and re-open it to get the admin link as long as you don't reload the page.

Reply:
OK, my suggestion from above does not make much sense, since the admin link is already "thrown at the user" in this case, which makes sense.
So, yes, `after reloading the website` is fine for me!